### PR TITLE
[needs sprites] Magic Mania 1.86: Life's a lich and then you die

### PR DESCRIPTION
**Pull request by Incoming5643**
Adds the phylactery item for wizards (NEEDS SPRITES). Using this item will allow wizards who fall in battle to revive from the phylactery as a lich (skeleton) two minutes after their death. This will continue until the phylactery is destroyed by the crew. 

While respawning wizards can see and speak with people, but can't cast spells or put up any sort of meaningful resistance. 

Liches spawn with robes and spells intact, but nothing else, everything collected on their original body stays there.

As a balance issue, the phylactery will not work from the den. 

Adds a nice black wizard robe ensemble for liches. d10_icon

Adds the phylactery to the rare prize pool for summon magic.

Modifies the round completion check for wizard mode to require two positive checks in a row, to allow for the momentary death lich wizards suffer before respawning in the phylactery. Also loosens the check from carbon wizards to living wizards, as a corgi with fireball is still very much a threat!

---

When the wizard dies, a message will be shown to those around that reveals the direction in which the phylactery lies.

Every time the wizard is ressurected, the time to ressurect gets one minute longer.

The phylactery must be kept on the same z-level as the wizard, if the wizard wanders off the z-level (or if the phylactery is thrown off the station) the wizard will be forcefully slain and recalled to the phylactery. A special check for the wizard den exists to make sure a wizard isn't ressurected there without any means to leave.

The phylactery binds on pickup now, as opposed to on attack_self

### CODE REVIEW

**Comment by Incoming5643** [comment]
Sprites needed, A general onfloor for the pendant/amulet that incorperates a gem somewhere on it as well as on mobs. I can probably make the on mobs, but I'm just so trash at designing.

---

Good places to keep the phylactery:

*On a trusted apprentice
*In the belly of the clown
*In the core of a subverted AI
*In a toliet

Bad places to keep the phylactery:
*On yourself
*In the den
*In the incinerator
*In the slime pits
*In the singularity

**Comment by Iamgoofball** [comment]
add a version of this known as soulguard:
1. only 1 use
2. sends you back to the den
3. uses up a spell slot
4. brings back as full human bean

balanced because 1 use unlike phylactery

**Comment by ike709** [comment]
can I keep it with Ian and trust him to guard it?

**Comment by Lumipharon** [comment]
Can you just be a dong and space it?

**Comment by AlexCrimson** [comment]
Seems like it would be pretty hard to find and destroy the thing if it can be placed pretty much anywhere. Make it a large item that cannot be moved when placed, and maybe limit it to the station z-level. Could also make the pinpointer track it.

**Comment by Incoming5643** [comment]
You can be a dong and space it, but skeletons in wizards robes are actually surprisingly spaceworthy.

File: code/modules/clothing/suits/wiz_robe.dm [hunk 1]
```diff
@@ -19,6 +19,11 @@
...
 	icon_state = "yellowwizard"
 
+/obj/item/clothing/head/wizard/black
+	name = "black wizard hat"
+	desc = "Strange-looking black hat-wear that most certainly belongs to a real magic user."
+	icon_state = "blackwizard"
+
 /obj/item/clothing/head/wizard/fake
 	name = "wizard hat"
 	desc = "It has WIZZARD written across it in sequins. Comes with a cool beard."
```
Comment (Iamgoofball) [inline code/modules/clothing/suits/wiz_robe.dm]: this should be on the sprite

**Comment by Lumipharon** [comment]
No you misunderstand. I mean can the wizard space it, so it becomes virtually impossible to destroy ever? The answer it seems, is yes.

**Comment by Incoming5643** [comment]
oh, yeah, I didn't quite think that one through. I might have to have it be a "only works if on same z level" thing

**Comment by Dorsisdwarf** [comment]
Make the phylactery respawn like the nuke disk if it goes off the Z-level.

Maybe make the pinpointer point to it while the wizard is regenerating from it.

**Comment by Chocobro** [comment]
![Image of Phylactery] (http://turd.church/ss13/shit-i-made/misc/phylactery.png)
http://turd.church/ss13/shit-i-made/misc/phylactery.dmi

**Comment by ExplosiveCrate** [comment]
The sprite should be a bit bigger, so you couldn't possibly miss it in a crowded room.

**Comment by Kromgar** [comment]
>The sprite isn't a urn full of blood and hair

**Comment by Kromgar** [comment]
So the wizard can pickup the phylactery right?

**Comment by Chocobro** [comment]
a phylactery is typically a charm or an amulet, so.

**Comment by Incoming5643** [comment]
Anyone can pick up the phylactery, and the wizard needs to attack_self it to start the protection anyways.

KEEPING the phylactery on your person once you're on station is a bad idea because it would obviously be found right away once you were killed

**Comment by phil235** [comment]
Is there any way for the crew to detect the phylactery outside of searching every corner of the station? That seems pretty much overpowered .

**Comment by Incoming5643** [comment]
Going to work on a method to give the crew a bit of a hint. Nothing as concrete as the pinpointer, but something to give them a fighting chance.

**Comment by Deantwo** [comment]
Nice idea, but the crew still need to be able to find it.

PinPointer?

**Comment by AlexCrimson** [comment]
It should be a bulky item that the wizard has to place, like a beacon or powersink. Having it as a small item just means the wizard can hid it in a box in a locker somewhere.

**Comment by Incoming5643** [comment]
looking at box I can say that for nearly anywhere a wizard could drop knowing a ordinal direction limits your search to one department and surrounding maint. You tell 30+ people "hey it's near/in medbay/engineering/sec" I have no doubts they could turn that place inside out in two minutes if they were organized.

I'd rather crank up the time to revive than make this into a "ungainly obvious impossible to hide item" thing.

**Comment by AlexCrimson** [comment]
Depends how much of the station he kills before they stop him. Even then it would probably take a few lives before you could pinpoints its location. Jaunt, Fireball, Teleport and a Phylactery hidden behind a false wall somewhere. Good luck finding that.

**Comment by Incoming5643** [comment]
Another downside for the wizard people probably aren't considering here: He can't take the escape shuttle/pods without retrieving or intentionally destroying his phylactery first.

**Comment by Incoming5643** [comment]
This pull is not dead, but probably comatose barring sprites and balance problems

**Comment by Incoming5643** [comment]
It'll be back some day, I promise

**Pull request closed without merging**